examples/classification/plot_classifier_comparison.py

```diff
@@ -123,10 +123,10 @@
         Z = Z.reshape(xx.shape)
         ax.contourf(xx, yy, Z, cmap=cm, alpha=.8)
 
-        # Plot also the training points
+        # Plot the training points in classifiers
```

Comment:
A bit confusing from my side. Maybe remove "in classifiers"?

Reply:
Thanks for reviewing. Have updated. :)